pages/notebook/index.tsx

```diff
@@ -0,0 +1,70 @@
+/* 
+  The main purpos of this file is to redirect user to their default
+  organization notebook.
+*/
+import nookies from "nookies";
+import NextError from "next/error";
+import { AUTH_TOKEN } from "~/config/constants";
+import { generateApiUrl } from "~/config/api";
+import { fetchUserOrgs } from "~/config/fetch";
+import { captureEvent } from "~/config/utils/events";
+
+interface Props {
+  errorCode: number;
+}
+
+const NotebookPage = ({ errorCode }: Props) => {
+  if (errorCode) {
+    return <NextError statusCode={errorCode} />;
+  }
+
+  return null;
+};
+
+export async function getServerSideProps(ctx) {
+  const cookies = nookies.get(ctx);
+  const authToken = cookies[AUTH_TOKEN];
+
+  // Ideally, we should redirect user to login page if they don't have an auth token
+  // However, because we do not have such a page, we will redirect them to 404
+  // Open issue: https://github.com/ResearchHub/researchhub-web/issues/1408
+  if (!authToken) {
+    return {
+      props: {
+        errorCode: 404,
+      },
+    };
+  }
```

Comment:
@lightninglu10 wanted to bring your attention to the above scenario ⬆️ 
LMK if you have thoughts about a standalone login page.

Reply:
I think the right thing to do here will be to wrap the login modal in a page. That should not be difficult.
If it's cool, I'd like to merge this and tackle the login modal separately.

Reply:
Actually, I'll just do it as part of this PR. I think it is okay.

Reply:
Done